spec/acceptance/native/backup_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'spec_helper_acceptance'
```

Comment:
It looks like all of the acceptance tests except `warn_header_spec.rb` were copied from spec/acceptance/ to spec/acceptance/native/ but the originals were not modified. Was this on purpose?